From b9f492491fd6349dd3f9dae627a16deeaa32c683 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 9 Jan 2024 14:19:55 +0100
Subject: [PATCH 0383/1695] dts: stm32: enable OSPI1 for stm32mp257f-ev1 board

Add the ommanager board and OSPI1 configuration on stm32mp257f-ev1 board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7c565b3da2ab25572c3edc04d6f020104af8338e dts: stm32: enable OSPI1 for stm32mp257f-ev1 board]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ibbdeaaf6bd1c3cebfe6227d9cbba4b761ee8a128
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/350017
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 6d6916d76..013621469 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -47,6 +47,21 @@
 	};
 };
 
+&ommanager {
+	memory-region = <&mm_ospi1>;
+	memory-region-names = "mm_ospi1";
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&ospi1_clk_pins_a
+		     &ospi1_io03_pins_a
+		     &ospi1_cs0_pins_a>;
+	pinctrl-1 = <&ospi1_clk_sleep_pins_a
+		     &ospi1_io03_sleep_pins_a
+		     &ospi1_cs0_sleep_pins_a>;
+	status = "okay";
+};
+
+
+
 &risaf2 {
 	status = "okay";
 };
-- 
2.47.2

